npm/npm_import.bzl

```diff
@@ -295,16 +329,43 @@ def npm_translate_lock(
             wild-cards are not yet supported and npm_translate_lock will fail if there are multiple versions of a package that
             are to be hoisted.
 
-        dev: If true, only install devDependencies
+            ```
+            public_hoist_packages = {
+                "@foo/bar": [""]
```

Comment:
add a comment about what empty string means (the root)

Reply:
✅